packages/workspace/src/utils/runtime-lint-utils.spec.ts

```diff
@@ -0,0 +1,190 @@
+import { checkCircularPath } from './runtime-lint-utils';
+
+function transformGraph(graph) {
+  const nodes = graph.nodes.reduce((acc, name) => {
+    return {
+      ...acc,
+      [name]: { name }
+    };
+  }, {});
+
+  const dependencies = Object.keys(graph.dependencies).reduce((acc, dep) => {
+    const targets = graph.dependencies[dep].reduce((acc, name) => {
+      return [...acc, { source: dep, target: name }];
+    }, []);
+
+    return {
+      ...acc,
+      [dep]: targets
+    };
+  }, {});
+
+  return {
+    nodes,
+    dependencies
+  };
+}
+
+function getPath(graph, node) {
+  return checkCircularPath(graph as any, { name: node.to } as any, { name: node.from } as any);
+}
+
+describe('should find the path between nodes', () => {
+  it('should return empty path when when there are no connecting edges', () => {
+    /*
+
+    A -> B -> C - > E
+
+     */
+
+    const graph = {
+      nodes: ['A', 'B', 'C', 'E'],
+      dependencies: {
+        'A': ['B'],
+        'B': ['C'],
+        'C': ['E']
+      }
+    };
+
+    const g = transformGraph(graph);
+    const path = getPath(g, { from: 'B', to: 'A' });
+
+    expect(path).toEqual([]);
+  });
+
+  it('should find direct path', () => {
+    /*
+
+    A -> B -> C - > E
+
+    */
+
+    const graph = {
+      nodes: ['A', 'B', 'C'],
```

Comment:
Same. I don't think C is needed, unless you think it changes the test case in a meaningful way